flink-container/src/test/java/org/apache/flink/container/entrypoint/StandaloneJobClusterEntryPointTest.java

```diff
@@ -59,6 +66,60 @@ public void testDontOverwriteExecutionMode() {
 		assertThat(getExecutionMode(configuration), equalTo(ExecutionMode.NORMAL));
 	}
 
+	@Test
+	public void configuredJobIDTakesPrecedenceWithHA() {
+		Optional<JobID> jobID = Optional.of(JobID.generate());
+
+		Configuration globalConfiguration = new Configuration();
+		globalConfiguration.setString(HighAvailabilityOptions.HA_MODE, "zookeeper");
+
+		JobID jobIdForCluster = StandaloneJobClusterEntryPoint.createJobIdForCluster(
+				jobID,
```

Comment:
Wrong indentation.

Reply:
Hopefully, done. Not completely sure, what the correct indentation is here, because my Flikn checkstyle profile actually produces this.